### DISCUSSION TITLE
paddleocr把中文安徽亳州识别成了毫州

### DISCUSSION THREAD

**Discussion by caishenzhishou** [PaddlePaddle/PaddleOCR#14077] [Q&A] [answered]
paddleocr把中文安徽亳州识别成了毫州，这个是字符集导致的吗？有啥优化的建议？

**Comment by caishenzhishou** [comment]
大佬们，有谁给个优化的建议？

**Comment by GreatV** [comment, marked as answer]
微调一下

**Reply by caishenzhishou** [reply]
哪里调？自己训练一下模型？

**Reply by Liyulingyue** [reply]
是的，需要自己训练一下

**Reply by caishenzhishou** [reply]
好的，多谢提醒

**Reply by caishenzhishou** [reply]
@Liyulingyue hello，能不能给个链接啥的，我学习下。我看网上太多了，感觉很乱啊。

**Reply by Liyulingyue** [reply]
@caishenzhishou 可以参考这个，目前文档可能仍有不足，但有同学在维护它，遇到问题欢迎反馈 https://paddlepaddle.github.io/PaddleOCR/latest/
